packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -221,23 +230,21 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
 
   /** @private */
   __calculateActive() {
-    const pathAbsolute = this.path.startsWith('/');
-    // Absolute path or no base uri in use. No special comparison needed
-    if (pathAbsolute) {
-      // Compare an absolute view path
-      return document.location.pathname === this.path;
+    if (!this.__pathExists(this.path)) {
```

Comment:
```suggestion
    if (path == null) {
```

Reply:
It should work, yes. I will replace them using this.

Reply:
Done.